public/javascripts/playlist/playlistTable.js

```diff
@@ -60,6 +77,10 @@ let playlistTable = {
   }
 };
 
+function editPlaylist(name, id) {
```

Comment:
Is this just for editing the playlist name?
If so, rename it to "editPlaylistName()"